File: projects/localserver/commons/src/main/java/com/robinfood/localserver/commons/dtos/orders/OrderDetailPaymentMethodDTO.java
package com.robinfood.localserver.commons.dtos.orders;

import lombok.AllArgsConstructor;
import lombok.Builder;
import lombok.Data;
import lombok.NoArgsConstructor;
import java.math.BigDecimal;

@AllArgsConstructor
@Builder
@Data
@NoArgsConstructor
public class OrderDetailPaymentMethodDTO {

    private Double discount;

    private Long id;

    private String name;

    private Long originId;

    private Double subtotal;

    private Double tax;

    private BigDecimal value;
}

File: projects/posv2/core/src/main/kotlin/com/robinfood/core/dtos/OrderDetailPaymentMethodDTO.kt
package com.robinfood.core.dtos

import java.math.BigDecimal

data class OrderDetailPaymentMethodDTO (
        val discount: BigDecimal,
        val id: Long,
        val originId: Long,
        val subtotal: BigDecimal,
        val tax: BigDecimal,
        val value: BigDecimal
)